presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -137,4 +139,28 @@ public GlueHiveMetastoreConfig setCatalogId(String catalogId)
         this.catalogId = Optional.ofNullable(catalogId);
         return this;
     }
+
+    public boolean isUseInstanceCredentials()
+    {
+        return useInstanceCredentials;
+    }
+
+    @Config("hive.metastore.glue.use-instance-credentials")
+    public GlueHiveMetastoreConfig setUseInstanceCredentials(boolean useInstanceCredentials)
+    {
+        this.useInstanceCredentials = useInstanceCredentials;
+        return this;
+    }
+
+    public Optional<String> getAwsCredentialsProvider()
+    {
+        return awsCredentialsProvider;
+    }
+
+    @Config("hive.metastore.glue.aws-credentials-provider")
+    public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsProvider)
+    {
+        this.awsCredentialsProvider = Optional.ofNullable(awsCredentialsProvider);
+        return this;
+    }
```

Comment:
use `@AssertTrue` to ensure that `use-instance-credentials` and `aws-credentials-provider` are not both set at the same time, see `io.prestosql.sql.analyzer.FeaturesConfig#isSpillerSpillPathsConfiguredIfSpillEnabled` for example